classes/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] Please add an `add_rental` method that is similar to this below to create an association with rentals. 
```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end

```

Reply:
Oh sure thank you for the suggestion. I am thinking to pass the whole rental object as parameters to `add_rental` function. Then I will not have to create the instance of `Rental` inside the method again